src/applications/vaos/new-appointment/components/CommunityCareProviderSelectionPage/RemoveProviderModal.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import Modal from '@department-of-veterans-affairs/formation-react/Modal';
+import { distanceBetween } from '../../../utils/address';
+
+export default function RemoveProviderModal({ onClose, provider, address }) {
+  const title = 'Are you sure you want to remove this provider?';
+  const content = (
+    <>
+      <span className="vads-u-display--block vads-u-font-weight--bold">
+        {provider.name}
+      </span>
+      <span className="vads-u-display--block">{provider.address?.line}</span>
+      <span className="vads-u-display--block">
+        {provider.address?.city}, {provider.address?.state}{' '}
+        {provider.address?.postalCode}
+      </span>
+      <span className="vads-u-display--block vads-u-font-size--sm vads-u-font-weight--bold">
+        {distanceBetween(
+          provider.position?.latitude,
+          provider.position?.longitude,
+          address.latitude,
+          address.longitude,
+        )}{' '}
+        miles
+      </span>
+      <button type="button" onClick={() => onClose(true)}>
+        Yes, remove provider
+      </button>
+      <button
+        className="usa-button-secondary"
+        type="button"
+        onClick={() => onClose(false)}
+      >
+        Cancel
+      </button>
+    </>
+  );
+
+  return (
+    <Modal
+      id="removeProviderModal"
+      visible
+      onClose={onClose}
+      title={title}
+      hideCloseButton
+    >
+      <div aria-atomic="true" aria-live="assertive">
```

Comment:
I don't think we need these two aria attributes, since this is in a modal and it doesn't need to alert a user beyond what a Modal already does.

Reply:
✔️